packages/auth/src/providers/cognito/apis/signInWithRedirect.ts

```diff
@@ -110,6 +111,11 @@ const oauthSignIn = async ({
 	// TODO(v6): use URL object instead
 	const oAuthUrl = `https://${domain}/oauth2/authorize?${queryString}`;
 
+	// this will be effective only on the following scenarios:
```

Comment:
Nit: Slight wording change to avoid confusion over "effectiveness"
```suggestion
	// this will only take effect in the following scenarios:
```